app/controllers/api/v1/accounts_controller.rb

```diff
@@ -21,13 +21,14 @@ def show
   end
 
   def create
-    token    = AppSignUpService.new.call(doorkeeper_token.application, account_params)
-    response = Doorkeeper::OAuth::TokenResponse.new(token)
+    not_found
+#    token    = AppSignUpService.new.call(doorkeeper_token.application, account_params)
+#    response = Doorkeeper::OAuth::TokenResponse.new(token)
 
-    headers.merge!(response.headers)
+#    headers.merge!(response.headers)
 
-    self.response_body = Oj.dump(response.body)
-    self.status        = response.status
+#    self.response_body = Oj.dump(response.body)
+#    self.status        = response.status
```

Comment:
Just do `not_found if ENV['HCAPTCHA_ENABLED'] == 'true'`

Reply:
Looking into this. Will a not_found interrupt the entire create action, or should I put the entirety of what I’ve commented out into a conditional block ?

Reply:
I think `not_found` interrupts the entire `create` action (it's a `respond_to`/`render` under the hood).
That said, thinking about it, I think it would be more in style with Mastodon's codebase to perform that check in a `before_action` (like for `check_enabled_registrations`).